src/airport.js

```diff
@@ -1 +1,98 @@
+class Airport {
 
+    constructor(airportID) {
+        this.airportID = airportID;
+        this.listOfPlanesAtAirport = [];
+        this.airportMaxCapacity = 2;
+        this.weatherSafeForAction = true;
+    }
+
+    get getAirportID() {
+        return this.airportID;
+    }
+
+    get getListOfPlanesAtAirport() {
+        return this.listOfPlanesAtAirport;
+    }
+
+    get getAirportMaxCapacity() {
+        return this.airportMaxCapacity;
+    }
+
+    setAirportMaxCapacity(newCapacity) {
+        if (newCapacity >= this.numberOfPlanesAtAirport() && typeof newCapacity === 'number' && newCapacity >= 0) {
```

Comment:
should have made this multi IF statement a separate method call with the three elements within it